RevenueCatUI/CustomerCenter/ManageSubscriptionsButtonStyle.swift

```diff
@@ -22,12 +23,17 @@ import SwiftUI
 @available(watchOS, unavailable)
 struct ManageSubscriptionsButtonStyle: ButtonStyle {
 
-    func makeBody(configuration: Configuration) -> some View {
+    var appearance: CustomerCenterConfigData.Appearance
+
+    @Environment(\.colorScheme)
+    private var colorScheme
+
+    func makeBody(configuration: ButtonStyleConfiguration) -> some View {
         configuration.label
             .padding()
             .frame(width: 300)
-            .background(Color.accentColor)
-            .foregroundColor(.white)
+            .background(color(from: appearance))
+            .foregroundColor(colorScheme == .dark ? Color.black : Color.white)
```

Comment:
Should this also come from the `Appearance`, maybe something like `onAccentColor`? 

If that's out of scope, maybe we can use an auto-adapting `Color` such as `primary`, instead of reading the `colorScheme`? 
```swift
.foregroundColor(Color.primary)
```

Reply:
I am going to change it to `Color.primary` for now

Reply:
oh, actually... `Color.primary` is the text color. So for light mode it is `black` and for dark mode `white`. For buttons we actually want the opposite